src/plugins/ncr/screens/node/AbstractDelegate.js

```diff
@@ -299,7 +299,8 @@ export default class AbstractDelegate {
     const node = this.component.props.getNodeRequestState.response.get('node');
     const formEvent = this.createFormEvent({}, { node, form: this.getFormName() });
     this.pbjx.trigger(formEvent.getMessage(), SUFFIX_INIT_FORM, formEvent);
-    return formEvent.getData();
+    // Force dirty is used for history revert.
+    return { ...formEvent.getData(), _forceDirty: false };
```

Comment:
@rsumilang i dont think is is necessary right, since dispatching it with the change will make it not pristine

Reply:
No it needs it because dispatching a change to a field that doesn't exist won't do anything.

Reply:
@rsumilang odd, it works fine for me when i just `return formEvent.getData();`

Reply:
@efjacobson hmm indeed... seems to work now after another sanity check